public/app/core/utils/richHistory.ts

```diff
@@ -5,7 +5,6 @@ import _ from 'lodash';
 import { DataQuery, DataSourceApi, dateTimeFormat, AppEvents, urlUtil, ExploreUrlState } from '@grafana/data';
```

Comment:
Change of SortOrder import as we have moved LogsSortOrder to grafana/data and moved SortOrder for Rich history to app/core/utils/richHistory.